_deploy_config.yml

```diff
@@ -0,0 +1,14 @@
+# get data from the hub
```

Comment:
I suggest renaming the file to `_config-deploy.yml` or `_config_deploy.yml`, for consistency and alphabetization's sake.